client/app/main/student/student.controller.js

```diff
@@ -2,18 +2,8 @@
 
 var app = angular.module('app');
 
-function StudentCtrl($scope, $stateParams, Intervention, Modal, Outreach,
-  Student, toastr) {
-
+function StudentCtrl($scope, $state, $stateParams, Student, toastr) {
   $scope.student = Student.get({id: $stateParams.id});
```

Comment:
There are 3 controllers being defined in this file now. The StudentCtrl doesn't use the Model service. It is being injected in the controller that does use it tho.